src/components/dev-hub/card-list.js

```diff
@@ -28,23 +30,58 @@ const HasMoreButtonContainer = styled('div')`
     text-align: center;
 `;
 
-export default React.memo(({ items = [], limit = 9 }) => {
+const getThumbnailUrl = media => {
+    return media.mediaType === 'twitch'
+        ? getTwitchThumbnail(media.thumbnailUrl, 1000)
+        : media.thumbnailUrl;
+};
+
+export default React.memo(({ videos = [], items = [], limit = 9 }) => {
     const [visibleCards, setVisibleCards] = useState(limit);
-    const hasMore = items.length > visibleCards;
+    const hasMore = videos.length
+        ? videos.length > visibleCards
+        : items.length > visibleCards;
+
     return (
         <>
-            <CardContainer>
-                {items.slice(0, visibleCards).map(item => (
-                    <ArticleCard
-                        to={item['slug']}
-                        key={item['_id']}
-                        image={withPrefix(item['atf-image'])}
-                        tags={getTagLinksFromMeta(item)}
-                        title={getNestedText(item['title'])}
-                        description={getNestedText(item['meta-description'])}
-                    />
-                ))}
-            </CardContainer>
+            {items.length > 0 && (
+                <CardContainer>
+                    {items.slice(0, visibleCards).map(item => (
+                        <ArticleCard
+                            to={item['slug']}
+                            key={item['_id']}
+                            image={withPrefix(item['atf-image'])}
+                            tags={getTagLinksFromMeta(item)}
+                            title={getNestedText(item['title'])}
+                            description={getNestedText(
+                                item['meta-description']
+                            )}
+                        />
+                    ))}
+                </CardContainer>
+            )}
+
+            {videos.length > 0 && (
+                <CardContainer>
+                    {videos.slice(0, visibleCards).map(video => (
+                        <VideoModal
+                            key={video.videoId}
+                            id={video.videoId}
+                            name={video.mediaType}
+                            trigger={
+                                <VideoCard
+                                    key={video['title']}
+                                    image={getThumbnailUrl(video)}
+                                    title={video['title']}
```

Comment:
```suggestion
                                    title={video.title}
```
since we don't have hyphens we can just use dot notation here and below. Even though it seems like we use it in the `ArticleCard` above, let's change all of the non-hyphen keys to dot notation (.slug, ._id, etc.) since that is our general style.